### PR TITLE
Update to RTCP routing rules

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/rtcweb-wg/jsep/issues/367

### CODE REVIEW

**Comment by fluffy** [comment]
I think the text Bernard is proposing here will it easier for implementors and also reduce interoperability problems.   I should to check the details but +1 on the general direction.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5874,13 +5902,27 @@ a=rtcp-fb:100 nack pli
...
-          packet to the "m=" line associated with that SSRC.</t>
-
-          <t>If the packet is of type RTPFB or PSFB, as defined in
-          <xref target="RFC4585" />, and the media source SSRC for the packet
-          is found in the outgoing SSRC table, deliver the packet
+          packet that is found in the incoming SSRC table, deliver a copy
+          of the packet to the "m=" line associated with that SSRC.</t>
+		
+	  <t>If the packet is an FIR (PT=PSFB, FMT=4), as defined in
+          <xref target="RFC5104" />, the SSRC found in each FCI entry
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: for consistency with the RR text above, suggest
> for each FCI entry where the SSRC is found in the outgoing SSRC table, deliver a copy of the RTCP packet to the "m=" line associated with that SSRC.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -5874,13 +5902,27 @@ a=rtcp-fb:100 nack pli
...
+          is matched against the outgoing SSRC table and a copy of the
+          packet is delivered to the "m=" lines associated with each
+          SSRC.</t>
+		
+	  <t>If the packet is an LRR (PT=PSFB, FMT=TBD) as defined in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: See above (although perhaps these paras could be combined?)

also, comma before 'as'

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -5874,13 +5902,27 @@ a=rtcp-fb:100 nack pli
...
+          <xref target="I-D.ietf-avtext-lrr" />, the SSRC found in each
+          FCI entry is matched against the outgoing SSRC table and a copy of
+          the packet is delivered to the "m=" lines associated with each SSRC.</t>
+		
+          <t>If the packet is of type RTPFB or type PSFB defined in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Do you need to exclude FIR/LRR here?

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (aboba) [inline draft-ietf-rtcweb-jsep.xml]: Done.

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (aboba) [inline draft-ietf-rtcweb-jsep.xml]: Combined the FIR and LRR paragraphs.

File: draft-ietf-rtcweb-jsep.xml (hunk 3 above)
Comment (aboba) [inline draft-ietf-rtcweb-jsep.xml]: Done.

**Pull request merged**